### PR TITLE
Nginx Site: tweaking fastcgi buffers

### PR DESCRIPTION
**Pull request by bryanlatten**
Also, allowing them to be easy tweaked at runtime

### CODE REVIEW

**Comment by ingluisjimenez** [comment]
👍

File: container/root/etc/nginx/sites-available/default [hunk 1]
```diff
@@ -37,6 +37,10 @@ server {
     # @see http://stackoverflow.com/questions/29770818/error-in-nginxphp-fpm-with-keepalive-fastcgi-keep-conn-on
     fastcgi_keep_conn off;
 
+    fastcgi_buffers 256 16k;
+    fastcgi_buffer_size 128k;
+    fastcgi_busy_buffers_size 256k;
```
Comment (ko-be) [inline container/root/etc/nginx/sites-available/default:42]: Double check w/ @bossjones on the latest settings @bryanlatten to see what's the most optimal that we've found so far.

Comment (bryanlatten) [reply container/root/etc/nginx/sites-available/default:42]: this is it. just promoting, and its available as a environment variable to enable continuous tweaking

**Comment by bossjones** [comment]
👍

**Comment by ko-be** [comment]
👍

**Pull request merged**